test/autodiff.in.cpp

```diff
@@ -42,7 +41,7 @@ TEST(${BACKEND_NAME}, backwards_maxpool_n4_c1_hw4_2x2_max)
 
     auto A = make_shared<op::Parameter>(element::i32, shape_a);
     auto reshape = make_shared<op::Reshape>(
-        A, AxisVector{0, 3, 1, 2}, Shape{1, 4, 4, 4}); //convert CHWN to CNHW
+        A, AxisVector{3, 0, 1, 2}, Shape{4, 1, 4, 4}); //convert CHWN to NCHW
```

Comment:
will this affect other backend. Or it was some bug?

Reply:
After discussing with @Krovatkin, we determined that the indices in the test were wrong (bug), but because the size of one of the dimensions was 1, the in-memory layout was equivalent. Since CNHW is not a common format, I believe the change I've made to the test was it's original intent (to NCHW). It shouldn't affect other backends.

Reply:
👍